views/teams/team.hbs

```diff
@@ -75,23 +75,23 @@
 
 
             <div class="course-card">
-                <div class="tabContainer">
+                <div class="tabContainer" active-tab={{activeTab}}>
```

Comment:
ich hab active-tab nicht als standard attribute gefunden. Eigene attribute sollten ein data- davor stehen haben.

Reply:
fixed in https://github.com/schul-cloud/schulcloud-client/pull/1235